backend/src/sharedCopy/applications.ts

```diff
@@ -39,6 +39,11 @@ class Application {
     const matches: string[] = result.match(/\[[^\]]*\]/g) || []
     return matches.map(m => m.slice(1, -1))
   }
+
+  launchBrowser(type: string) {
+    const platform = navigator ? navigator.platform.toLowerCase() : ''
+    return !(type === 'SSH' && platform.includes('win'))
```

Comment:
There is a `Browser.ts` service file that has an isWindows() method already you can use here.

```suggestion
    return !(type === 'SSH' && isWindows())
```

Reply:
`isWindows()` is going to broke the copyShare in backend